util.py

```diff
@@ -146,7 +152,7 @@ def push_robot(self, robot, state):
         # PusherTile property game_round:
         #  0 for even game round number,
         #  1 for odd game round number.
-        if state.game_round % 2 and self.game_round:
+        if state.game_round % 2 == self.game_round:
```

Comment:
Řádky 155 a 158 jsou teď úplně stejné. Opravdu ověřují to, co se zde musí vyhodnotit?

Reply:
A jo, to jsem tam zapomněla smazat. Díky ;)